style-guide/styles/wds-mixins/_wds-dropdown-linked-list.scss

```diff
@@ -1,4 +1,4 @@
 @mixin wds-dropdown-linked-list-item-hover {
-	background-color: fade_out($wds-color-link, 0.9);
-	color: $wds-color-link;
+	background-color: fade_out($wds-fandom-color-aqua, 0.9);
```

Comment:
@latata I think that this will change dropdown theming everywhere, not only in global nav. Do we want this?